src/com/ebcitron/piggybank/Coin.java

```diff
@@ -0,0 +1,22 @@
+package com.ebcitron.piggybank;
+
+public abstract class Coin {
+
+     double value;
+     double quantity = 1;
+
+
+    public Coin(double quantity) {
+        this.quantity = quantity;
+    }
+
+    public Coin() {
```

Comment:
It may make sense to initialize quantity to 1 if you don't pass it in to the constructor